README.md

```diff
@@ -1,28 +1,29 @@
 [![GitHub release](https://img.shields.io/github/release/cybozu-go/pod-security-admission.svg?maxAge=60)][releases]
-[![CircleCI](https://circleci.com/gh/cybozu-go/pod-security-admission.svg?style=svg)](https://circleci.com/gh/cybozu-go/pod-security-admission)
+![CI](https://github.com/cybozu-go/pod-security-admission/workflows/CI/badge.svg)
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/pod-security-admission?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/pod-security-admission?tab=overview)
 [![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/pod-security-admission)](https://goreportcard.com/report/github.com/cybozu-go/pod-security-admission)
 
 Pod Security Admission
 ======================
 
-**Project Status**: Initial development
+**Project Status**: Under development
 
-`pod-security-admission` is composed of [validation and mutating admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) to ensure [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+pod-security-admission is a [Kubernetes Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) to ensure [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
```

Comment:
As far as I know, Pod Security Admission is not a single unit of webhook, but a set of validating and mutating admission webhooks. So "a set of" or "a pack of" sounds more natural to me.